app.js

```diff
@@ -15,31 +15,70 @@ const winsEl = document.getElementById('wins');
 const hidingPlaces = ['tree', 'shed', 'boulder'];
 
 let correctGuesses = 0;
+let incorrectGuesses = 0;
 let totalGuesses = 0;
 
 shedButton.addEventListener('click', () => {
     const hidingSpot = Math.floor(Math.random() * 3);
     const answer = hidingPlaces[hidingSpot];
     handleGuess(answer, 'shed');
+    console.log('the hiding spot is', hidingSpot);
+
+    
 });
 
 treeButton.addEventListener('click', () => {
     const hidingSpot = Math.floor(Math.random() * 3);
     const answer = hidingPlaces[hidingSpot];
     handleGuess(answer, 'tree');
+    console.log('the hiding spot is', hidingSpot);
 });
 
 boulderButton.addEventListener('click', () => {
     const hidingSpot = Math.floor(Math.random() * 3);
     const answer = hidingPlaces[hidingSpot];
     handleGuess(answer, 'boulder');
+    console.log('the hiding spot is', hidingSpot);
 });
 
 function handleGuess(correctSpot, userGuess) {
     // reset the styles
+    shedContainer.classList.remove('face');
+    treeContainer.classList.remove('face');
+    boulderContainer.classList.remove('face');
+
     // then increment the guesses
+    if (handleGuess === true || handleGuess === false) {
+        totalGuesses++;
+    }
+
     // then grab the appropriate container element for the correct guess from the DOM
+    document.getElementById(`${correctSpot}-container`).classList.add('face');
     // then add the face class to that element so that the face shows up
+    if (correctSpot === 'shed') {
+        shedContainer.classList.add('face');
+        treeContainer.classList.remove('face');
+        boulderContainer.classList.remove('face');
+    } else if (correctSpot === 'tree') {
+        treeContainer.classList.add('face');
+        shedContainer.classList.remove('face');
+        boulderContainer.classList.remove('face');
+    } else {
+        boulderContainer.classList.add('face');
+        treeContainer.classList.remove('face');
+        shedContainer.classList.remove('face');
+    }
+
     // then if the user guess is correct, increment the correct guesses
-    // update the DOM to show this change to the user (including the losses, not tracked directly in state)
-}
+    if (userGuess === true) {
```

Comment:
userGuess will always be a string of 'boulder', 'shed', or 'tree' - so checking if it equals true is causing your correct number to not be incremented. It'd be helpful to think about what the "winning" scenario means considering the arguments you're passing into your function (userGuess & correctSpot)